src/renderer/fonts/font_options.rs

```diff
@@ -59,17 +65,30 @@ impl FontOptions {
     }
 
     pub fn get_weight(&self, bold: bool) -> FontWeight {
-        if self.bold || bold {
+        if bold {
             FontWeight::Bold
```

Comment:
Here I've decided not to use `self.bold` when getting `FontWeight` (or `FontSlant` below) from `FontOptions`, so the code can be reused at sections where `FontOptions` bold option should not be taken into account (for example, here at [CachingShape](https://github.com/neovide/neovide/pull/869/commits/fbe0c3d8da8ad4bd84b4688631d3393ba2a44bb4#diff-ab22eb44387b5cd97300475e15e02e47e8725088ccbfeebc7fe99cac0a8d9bc2L180-R171))

Reply:
That's good idea.